### DISCUSSION TITLE
Optional Article Summary

### DISCUSSION THREAD

**Discussion by vavramovski** [dailydotdev/daily#651] [Feature Requests]
I don't like the new change when I click on an article, I don't want to read the summary of the article. 
I want to be able to open multiple articles and then read them. 
This new feature slows me down.

**Comment by idoshamun** [comment]
Hi @vavramovski,

We're currently testing this new behavior. It's not completely rolled out.
Once and if we apply it to everyone, we might introduce a setting to change it back. Still in discussion, though.
With that said, you have the button "Read article" which acts exactly like the old version, so you can open as many articles as you want.
